CMakeLists.txt

```diff
@@ -287,15 +287,13 @@ add_custom_target(docs DEPENDS "${re2c_docs}")
 # install and test targets are enabled only if re2c is the root project
 if (CMAKE_SOURCE_DIR STREQUAL PROJECT_SOURCE_DIR)
     # install
-    if(UNIX)
-        install(TARGETS re2c RUNTIME DESTINATION bin)
-        install(FILES "${re2c_manpage_c}" DESTINATION "share/man/man1")
-        if(RE2C_BUILD_RE2GO)
-            install(TARGETS re2go RUNTIME DESTINATION bin)
-            install(FILES "${re2c_manpage_go}" DESTINATION "share/man/man1")
-        endif()
-        install(FILES include/unicode_categories.re DESTINATION "${RE2C_STDLIB_DIR}")
+    install(TARGETS re2c RUNTIME DESTINATION bin)
+    install(FILES "${re2c_manpage_c}" DESTINATION "share/man/man1")
+    if(RE2C_BUILD_RE2GO)
+        install(TARGETS re2go RUNTIME DESTINATION bin)
+        install(FILES "${re2c_manpage_go}" DESTINATION "share/man/man1")
     endif()
+    install(FILES include/unicode_categories.re DESTINATION "${RE2C_STDLIB_DIR}")
```

Comment:
To me this change seems correct, but when the question of Windows install rules came up in a discussion some time ago (https://github.com/skvadrik/re2c/issues/302#issuecomment-676672529), there were concerns about default install paths (which I don't understand, but I don't have Windows).

It works in the CI environment ([here](https://github.com/skvadrik/re2c/pull/319/checks?check_run_id=1247973854)), but uses the POSIX file hierarchy with forward slashes:
```
  -- Install configuration: "Debug"
  -- Installing: D:/a/re2c/re2c/install/bin/re2c.exe
  -- Installing: D:/a/re2c/re2c/install/share/man/man1/re2c.1
  -- Installing: D:/a/re2c/re2c/install/bin/re2go.exe
  -- Installing: D:/a/re2c/re2c/install/share/man/man1/re2go.1
  -- Installing: D:/a/re2c/re2c/install/share/re2c/stdlib/unicode_categories.re
```
And the choice of `D:/a/re2c` prefix is unclear to me: where does it come from?

On the whole, I'd rather enable install rules on Windows and wait until someone complains, but I'm far from understanding how the Windows environment works.

Reply:
`D:/a/re2c/re2c` is a project directory inside CI environment . I have Windows and I see the same result / output

Reply:
I think the default prefix on Windows is `c:/Program Files/${PROJECT_NAME}`, at least that's what [CMake documentation](https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_PREFIX.html) says. I'm not suggesting to override it.